### DISCUSSION TITLE
Playing multiple files in succession without a playlist

### DISCUSSION THREAD

**Discussion by gkovac42** [alexmercerind/dart_vlc#244] [Q&A]
May I ask why was my issue closed without an answer, and in such a rude manner @alexmercerind ? https://github.com/alexmercerind/dart_vlc/issues/243

Sorry if I offended you in any way, I'm just trying to understand how to play multiple files in fast succession without causing lag spikes...

I did not spam, I just provided you with a recording of the issue, and I also said that performance mode makes no difference...

**Comment by alexmercerind** [comment]
Use the `Playlist` constructor.

[#open-a-media-or-playlist-into-a-player](https://github.com/alexmercerind/dart_vlc#open-a-media-or-playlist-into-a-player)

```dart
final playlist = Playlist(
  medias: [
    Media.file(File('C:/music.mp3')),
    Media.file(File('C:/audio.mp3')),
    Media.asset('assets/audio/example.mp3'),
    Media.network('https://www.example.com/music.aac'),
  ],
);
player.open(playlist);
```

**Comment by gkovac42** [comment]
As stated in the title, I was asking if it is possible to do it without a playlist. I want to play a bunch of different files on different button clicks, and keeping all files in a playlist is not really ideal. Just wanted to know if it can be done or not.

**Reply by alexmercerind** [reply]
> As stated in the title, I was asking if it is possible to do it without a playlist. I want to play a bunch of different files on different button clicks, and keeping all files in a playlist is not really ideal. Just wanted to know if it can be done or not.

Use `player.add`, `player.remove`, `player.insert`, `player.jumpToIndex`, `player.next`, `player.previous`. Don't try to implement playlist logic entirely in Dart.

**Reply by gkovac42** [reply]
Ok, I will try that. Thanks for answering and sorry if I upset you, it really wasn't my intention.
